### ISSUE TITLE
Add Keyboard-Only Controls for Card Selection and Game Actions

### ISSUE THREAD

**Issue by Zunon** [Cockatrice/Cockatrice#5043]
**Similar Requests**

None found.

**Description of New Feature**

I would like to request the implementation of a feature that allows users to play on Cockatrice entirely using the keyboard, without needing a mouse. Currently, the main challenge is selecting cards using only keyboard shortcuts. Introducing a set of keyboard shortcuts for card selection and any other game actions that aren't covered would greatly enhance accessibility and usability for keyboard-only users.

**Additional Context**

This feature would be particularly beneficial for users with limited mobility or those who prefer using the keyboard over the mouse. It would also improve the overall user experience by providing an alternative control method.

**Comment by ebbit1q** [comment]
actions that aren't available without mouse:
- selecting cards
- drawing arrows
- moving cards in play
- moving cards to and from sideboard before starting games

there are multiple ways we could implement selection behavior:
- using a modifier key: holding a modifier key eg ctrl with a movement key like the arrow keys moves the selection across the field
- modal: using a keybind enters a selection mode in which the selection can be moved and expanded
- keyboard mode toggle in settings: after enabling it in the settings the selection is always moved using the keyboard shortcuts selected in the normal shortcut menu or a special one
- some combination, I think it's important that this addition does not affect existing users too much

currently cockatrice can be fully used using just a mouse, it'd be nice to have the same kind of parity with a keyboard, as an addition it might be nice to also use the same right click menus as the mouse has with the keyboard in case you don't want to maintain a list of all keybinds for every action

**Comment by Censor-Array** [comment]
Seems like the arrow keys would be a good way to hop between select-able items using the same gui effect as a mouse-over. The app button could open the right-click menu. But how would we swap between windows to select something in the graveyard or library?
